gamemode/sh_disguise.lua

```diff
@@ -111,4 +111,33 @@ end
 
 function PlayerMeta:DisguiseRotationLocked()
 	return self:GetNWBool("disguiseRotationLock")
+end
+
+function GM:PlayerCanDisguiseCurrentTarget(ply)
+	if !IsValid(ply) then return false, nil end
+	
+	local HorizLeniency = 50
```

Comment:
Similarly here; I prefer these to be lowercased to match the rest of the codebase.